src/Api/PubnubApi/Security/Crypto/CryptoModule.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using PubnubApi.Security.Crypto.Cryptors;
+
+namespace PubnubApi.Security.Crypto
+{
+    public class CryptoModule
+    {
+        private readonly ICryptor _cryptor;
```

Comment:
The module should allow for multiple cryptors

Reply:
This module supports multiple cryptors using constructor. First constructor expects new or old cryptor(due to contract ICryptor) and second one LegacyCryptor type.
Current we have only two cryptors.
This module supports any other cryptor as well as long as it implements the contract and pass its type into constructor.

Reply:
refactored to support multiple cryptors.